app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -716,6 +717,10 @@ export default function OnboardingSetup({
   const regionPopulationYear = watch("regionPopulationYear");
   const countryPopulationYear = watch("countryPopulationYear");
 
+  const { data: cityArea, isLoading: isCityAreaLoading } = api.useGetCityBoundaryQuery(data.locode!, {
+    skip: !data.locode,
+  });
```

Comment:
We are querying this route anyways to display the map. Can you just use the area data that's being provided from the map outline endpoint?

Reply:
And I think you need to destructure the data property to the the cityArea and map/ bounds here